bigquery/calculate-daily-vacancy-tag-time-metrics.sql

```diff
@@ -84,7 +71,7 @@ WITH
   FROM
     dates
   CROSS JOIN
-    scraped_vacancies_in_scope
+     `teacher-vacancy-service.production_dataset.scraped_vacancies_in_scope` AS scraped_vacancies_in_scope
```

Comment:
Is this the name of a table/column? Will this table/column get out of date quickly and will we forget that it is out of date? Worth putting in a comment?